cmd/kubectl-gadget/bcck8s.go

```diff
@@ -97,18 +95,51 @@ func init() {
 		tcptracerCmd,
 		capabilitiesCmd,
 	}
-	args := []string{"label", "node", "namespace", "podname", "containername"}
-	vars := []*string{&labelParam, &nodeParam, &namespaceParam, &podnameParam, &containernameParam}
+
+	// Add flags for all BCC gadgets
 	for _, command := range commands {
 		rootCmd.AddCommand(command)
-		for i, _ := range args {
-			command.PersistentFlags().StringVar(
-				vars[i],
-				args[i],
-				"",
-				fmt.Sprintf("Kubernetes %s selector", args[i]))
-		}
+		command.PersistentFlags().StringVarP(
+			&labelParam,
+			"selector",
+			"l",
+			"",
+			fmt.Sprintf("Labels selector to filter on. Only '=' is supported.(e.g. key1=value1)."),
```

Comment:
`s/.(/ (/`

Reply:
You could add two labels in the example to show they are separated by `,`:
`-l key1=value1,key2=value2`

Reply:
Done!